scripts/us_bea/states_gdp/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "USStatesQuarterlyGDP",
+            "curator_emails": [
+                "fpernice@google.com"
+            ],
+            "provenance_url": "https://www.bea.gov/data/gdp/gdp-state",
+            "provenance_description": "Quarterly data on inflation-adjusted GDP at a state level in the United States from the US Bureau of Economic Analysis.",
+            "scripts": [
+                "import_data_test.py",
+                "validate_import.py",
```

Comment:
I think you might want to leave out `validate_import.py`. That one validates the data by downloading a bunch of stuff, so it'll take some time to run. `import_data_test.py` is a more traditional set of tests.

Reply:
resolved per discussion